src/assets/metadata/aws.json

```diff
@@ -1,296 +1,3731 @@
+              "description": "The port the load balancer uses when performing health checks on targets.",
+              "displayName": "Port",
+              "name": "port",
+              "type": "Number",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#port"
+            },
+            {
+              "description": "The port the load balancer uses when performing health checks on targets.",
+              "displayName": "Protocol",
+              "name": "protocol",
+              "rules": {
+                "values": [
+                  "TCP",
+                  "HTTP",
+                  "HTTPS"
+                ]
+              },
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#protocol"
+            },
+            {
+              "description": "Amount of time, in seconds, during which no response from a target means a failed health check.",
+              "displayName": "Timeout",
+              "name": "timeout",
+              "type": "Number",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#timeout"
+            },
+            {
+              "description": "Number of consecutive health check failures required before considering a target unhealthy.",
+              "displayName": "Unhealthy threshold",
+              "name": "unhealthy_threshold",
+              "type": "Number",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#unhealthy_threshold"
+            }
+          ],
+          "description": "Health Check configuration block.",
+          "displayName": "Health check",
+          "name": "health_check",
+          "type": "Object",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#health_check"
+        },
+        {
+          "attributes": [
+            {
+              "description": "The time period, in seconds, during which requests from a client should be routed to the same target.",
+              "displayName": "Cookie duration",
+              "name": "cookie_duration",
+              "type": "Number",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#cookie_duration"
+            },
+            {
+              "description": "Name of the application based cookie.",
+              "displayName": "Cookie name",
+              "name": "cookie_name",
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#cookie_name"
+            },
+            {
+              "description": "Boolean to enable/disable stickiness.",
+              "displayName": "Enabled",
+              "name": "enabled",
```

Comment:
Could it be more specific ? `Enable stickiness`

Reply:
done